projects/ngrx-auto-entity/src/index.ts

```diff
@@ -103,62 +96,57 @@ export {
   EndEdit,
   EditEnded,
   Clear
-} from './lib/actions';
-
-export {
-  IEffectExclusions,
-  IEffectExcept,
-  IEntityOptions,
-  IEntityTransformer,
-  Entity,
-  all,
-  extra,
-  loads,
-  curd,
-  except,
-  matching
-} from './lib/decorators/entity';
+} from './lib/actions/actions';
 
+export { ENTITY_OPTS_PROP } from './lib/decorators/entity-tokens';
+export { IEffectExcept, IEntityOptions, IEntityTransformer, Entity } from './lib/decorators/entity';
+export { IEffectExclusions } from './lib/decorators/effect-exclusions';
+export { curd } from './lib/decorators/effect-exclusion-utils';
+export { loads } from './lib/decorators/effect-exclusion-utils';
+export { extra } from './lib/decorators/effect-exclusion-utils';
+export { all } from './lib/decorators/effect-exclusion-utils';
+export { matching } from './lib/decorators/effect-exclusion-utils';
+export { except } from './lib/decorators/effect-exclusion-utils';
```

Comment:
Are these separate exports intended or should they be grouped?

Reply:
They should be grouped. I think this is a consequence of the extract to file process in webstorm.